packages/terra-table/tests/jest/__snapshots__/ColumnHeaderCell.test.jsx.snap

```diff
@@ -80,6 +80,7 @@ exports[`ColumnHeaderCell renders a pinned column header cell 1`] = `
     width={100}
   >
     <th
+      aria-sort="ascending"
```

Comment:
If we do have to add this attribute, there should be a larger impact to testing.

Reply:
Note: No longer doing this change after it was advised to not use aria-sort.